README.md

```diff
@@ -55,6 +56,19 @@ this variable, identical to default one.
 When the platform is OpenBSD, the variable has a single key, `flags` and its
 value is empty string.
 
+## `uchiwa_include_role_x509_certificate`
+
+When `yes`, this variable includes and execute
+[`reallyenglish.x509`](https://github.com/reallyenglish/ansible-role-x509-certificate)
+during the play, which makes it possible to manage certificates without ugly
+hacks. This is only supported in `ansible` version _at least_ 2.2 and later.
+
+Known supported platforms:
+
+| platform | `ansible version` |
+|----------|-------------------|
+| CentOS   | 2.3.1.0           |
```

Comment:
s/CentOS/RedHat/ ?

FreeBSD has the official package of 2.3.1.0 and Ubuntu has a ppa https://launchpad.net/~ansible/+archive/ubuntu/ansible.